CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
+* Fix strucutured data for breadcrumbs with no URL.
```

Comment:
`- Correct breadcrumb structured data for missing URL (PR #180)`

Reply:
I'm sure that's what I meant